sdk/identity/azure-identity/src/client_certificate_credential.cpp

```diff
@@ -143,12 +143,15 @@ std::string GetJwtToken(
   std::string x5cHeaderParam{};
   if (sendCertificateChain)
   {
-    // Since there is only one base64 encoded cert string, it can be written as a JSON string rather
-    // than a JSON array of strings.
-    x5cHeaderParam = ",\"x5c\":\"";
+    // Even though there is only one base64 encoded cert string, we write the x5c header as a JSON
+    // array of strings, following the spec:
+    // https://datatracker.ietf.org/doc/html/rfc7517#section-4.7
+    // For historical, legacy reasons, the auth endpoint happens to work with a single string, but
```

Comment:
```suggestion
    // For historical, legacy reasons, the token endpoint happens to work with a single string, but
```
Just a nit. Otherwise, LGTM.